pkg/dss/scd/models/operations.go

```diff
@@ -35,3 +36,39 @@ type Operation struct {
 	Cells          s2.CellUnion
 	SubscriptionID ID
 }
+
+// AdjustTimeRange adjusts the time range to the max allowed ranges on an
+// Operation.
+func (o *Operation) AdjustTimeRange(now time.Time, old *Operation) error {
+	if o.StartTime == nil {
+		// If StartTime was omitted, default to Now() for new ISAs or re-
+		// use the existing time of existing ISAs.
+		if old == nil {
+			o.StartTime = &now
+		} else {
+			o.StartTime = old.StartTime
+		}
+	} else {
+		// If setting the StartTime explicitly ensure it is not too far in the past.
```

Comment:
The start time for Operations can be in the past

Reply:
Done.